ert3/workspace/_workspace.py

```diff
@@ -25,6 +35,8 @@ def _locate_root(path: Union[str, Path]) -> Optional[Path]:
 def assert_experiment_exists(
     workspace_root: Union[str, Path], experiment_name: str
 ) -> None:
+    """Check if the experiment `experiment_name` exists in workspace."""
+
```

Comment:
Is there a way to have the function signatures generated, even though there are no docstrings defined? I think it would be nice to at least show this info always 🙂